daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -953,3 +975,68 @@ class DaffodilXMLReader private[japi] (xmlrdr: SDaffodilXMLReader) extends org.x
    */
   def parse(arr: Array[Byte]): Unit = xmlrdr.parse(arr)
 }
+
+/**
+ * Read in an infoset from any SAX Reader for unparsing
+ */
+class DaffodilInputContentHandler private[japi] (sContentHandler: SDaffodilInputContentHandler)
+  extends ContentHandlerProxy {
+  override val contentHandler: org.xml.sax.ContentHandler = sContentHandler
+
+  def getUnparseResult: UnparseResult =
+    new UnparseResult(sContentHandler.getUnparseResult.asInstanceOf[SUnparseResult])
+  def enableInputterBlobUriAbsolutization(): Unit= sContentHandler.enableInputterBlobUriAbsolutization()
```

Comment:
Needs scala doc, and should mention that this should note be used in production. I'm also not sure about "Absolutization". Maybe something like "resolveRelativeInfosetBlobURIs()

Reply:
Decided to take it out, since this is used specifically for TDML. Also renamed!